test/node.js

```diff
@@ -37,6 +41,94 @@ test('node pass on nodeOptions', async t => {
 	t.is(stdout, 'foo');
 });
 
+test.serial(
+	'node removes --inspect from nodeOptions when defined by parent process',
+	async t => {
+		const originalArgv = process.execArgv;
+		process.execArgv = ['--inspect', '-e'];
+		const {stdout, stderr} = await execa.node('console.log("foo")', {
+			stdout: 'pipe'
```

Comment:
`stdout` defaults to `pipe`, so does not need to be defined here (and in the following tests).